packages/composite-checkout/src/components/field.js

```diff
@@ -72,7 +72,7 @@ Field.propTypes = {
 	className: PropTypes.string,
 	isError: PropTypes.bool,
 	onChange: PropTypes.func,
-	label: PropTypes.string.isRequired,
+	label: PropTypes.string,
```

Comment:
Minor, but if we don't have a label, maybe we should not display the `Label` at all? That is, maybe change line 44 to be `{label && <Label htmlFor={ id }>{ label }</Label>}`?

Reply:
Good call. I didn't notice it rendering without the label being passed. I'll update this.